hathor/indexes/timestamp_index.py

```diff
@@ -32,6 +50,20 @@ class TimestampIndex(BaseIndex):
     """ Index of transactions sorted by their timestamps.
     """
 
+    def __init__(self, *, txs: bool = False, blocks: bool = False, all: bool = False):
+        if sum([txs, blocks, all]) != 1:
+            raise TypeError('Exactly one of these: txs, blocks, all, must be set to True')
+        self._scope: Scope
+        if txs:
+            self._scope = SCOPE_TXS
+        elif blocks:
+            self._scope = SCOPE_BLOCKS
+        elif all:
+            self._scope = SCOPE_ALL
```

Comment:
Same as in `hathor/indexes/tips_index.py`

Reply:
Done.